### PR TITLE
validation for page_no and fix presentation url .

### PR DESCRIPTION
**Pull request by ziadhany**
add validation for page_no #610 with a unit test , black format .
 and fix presentation url .

### CODE REVIEW

**Comment by Hritik14** [comment]
@ziadhany As the development in this branch is going on and we wanted to move ahead with #476, rebasing/merging this with/from main will likely cause merge conflicts. Please accept the incoming changes for the import statements.
Alternatively, give [kdiff3](http://kdiff3.sourceforge.net/) a try for solving merge conflicts, it's really smart.

**Comment by pombredanne** [comment]
@ziadhany I think we are good now...
May be you could squash your commits?
Thanks!

File: README.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ compromise.
 
 Recent presentations:
 
-- `Open Source Summit 2020 <docs/Why-Is-There-No-Free-Software-Vulnerability-Database-v1.0.pdf>`__
+- `Open Source Summit 2020 <docs/Presentations/Why-Is-There-No-Free-Software-Vulnerability-Database-v1.0.pdf>`__
```
Comment (pombredanne) [inline README.rst:80]: Just a nit... this is OK for now, but in general you want to avoid unrelated changes in a pull request.

Comment (ziadhany) [reply README.rst:80]: OK ,  I'll take that into consideration next time.

**Review by pombredanne — CHANGES_REQUESTED** [review]
All good. Thanks!
